bin/vyper

```diff
@@ -41,7 +41,8 @@ if __name__ == '__main__':
 
     # Combined json output
     if args.format == 'combined_json':
-        out = compiler.compile(codes, ['bytecode', 'bytecode_runtime', 'abi', 'source_map'], output_type='dict')
+        out = vyper.compile(codes, ['bytecode', 'bytecode_runtime', 'abi', 'source_map'], output_type='dict')
```

Comment:
I was picturing that the `vyper.compile` interface would only take a single source program and compile it, not multiple. Can multiple files be handled here in this script instead?

e.g.
```python
out = {}
for filename, code in codes.items():
    out[filename] = vyper.compile(code, returns=['bytecode', 'bytecode_runtime', 'abi', 'source_map'])
```

Reply:
Yes but then we loose capability of handling multiple inputs on the vyper module. That's why I chose to make the vyper binary skinny instead.